lib/spack/spack/test/compilers/basics.py

```diff
@@ -465,6 +465,28 @@ def test_aocc_flags():
     supported_flag_test("fc_pic_flag", "-fPIC", "aocc@2.2.0")
     supported_flag_test("version_argument", "--version", "aocc@2.2.0")
 
+    supported_flag_test("debug_flags",
+                        ['-gcodeview', '-gdwarf-2', '-gdwarf-3',
+                         '-gdwarf-4', '-gdwarf-5', '-gline-tables-only',
+                         '-gmodules', '-gz', '-g'],
+                        'aocc@2.3.0')
+    supported_flag_test("opt_flags",
+                        ['-O0', '-O1', '-O2', '-O3', '-Ofast',
+                         '-Os', '-Oz', '-Og',
+                         '-O', '-O4'],
+                        'aocc@2.3.0')
+    supported_flag_test("openmp_flag", "-fopenmp", "aocc@2.3.0")
+    supported_flag_test("cxx11_flag", "-std=c++11", "aocc@2.3.0")
+    supported_flag_test("cxx14_flag", "-std=c++14", "aocc@2.3.0")
+    supported_flag_test("cxx17_flag", "-std=c++17", "aocc@2.3.0")
+    supported_flag_test("c99_flag", "-std=c99", "aocc@2.3.0")
+    supported_flag_test("c11_flag", "-std=c11", "aocc@2.3.0")
+    supported_flag_test("cc_pic_flag", "-fPIC", "aocc@2.3.0")
+    supported_flag_test("cxx_pic_flag", "-fPIC", "aocc@2.3.0")
+    supported_flag_test("f77_pic_flag", "-fPIC", "aocc@2.3.0")
+    supported_flag_test("fc_pic_flag", "-fPIC", "aocc@2.3.0")
+    supported_flag_test("version_argument", "--version", "aocc@2.3.0")
+
```

Comment:
If there are no changes in flags from v2.2.0 these can be omitted.

Reply:
Will it cause any coverage issue ?

Reply:
@GaneshPrasadMA No, since you didn't add any additional lines - nor you are testing any additional behavior as far as I can see.

Reply:
okay, thanks

Reply:
Hi @alalazo ,

I have addressed your above review comments